web/viewer.css

```diff
@@ -1701,6 +1701,8 @@ html[dir="rtl"] #documentPropertiesOverlay .row > * {
 }
 #PDFBug table {
   font-size: 10px;
+  font-family: monospace;
```

Comment:
I'm finding that the use of a monospace font makes it harder to read the text comfortably, at least on Windows.

Do we *really* need to change the font-family here, since it seems to me that the indentation is still noticeable even without that?
Another possible idea, to improve readability of the monospace font, could be to increase the font-size slightly? (Although that would require even more horizontal scrolling during reading.)